src/Migration/Migration1675858542AddEnderecoAmsStatusColumn.php

```diff
@@ -20,9 +20,9 @@ public function update(Connection $connection): void
     {
         $sql = <<<SQL
         ALTER TABLE `endereco_address_ext`
-            ADD COLUMN `ams_status` LONGTEXT NOT NULL DEFAULT 'not-checked' AFTER `address_id`,
+            ADD COLUMN `ams_status` LONGTEXT NULL AFTER `address_id`,
```

Comment:
Before I test it, quick question. Does your logic rely on "not-checked" default value? If its NULL or empty string instead, can you foresee any problems?